### ISSUE TITLE
Impossible to install Blocktrail SDK with Laravel 5.2

### ISSUE THREAD

**Issue by adaur** [blocktrail/blocktrail-sdk-php#28]
Hello,

I am trying to install your SDK to work with my fresh Laravel installation. Composer cannot add it as a dependency because of the "illuminate/cookie" package that requires "symfony/http-foundation": "2.8.*|3.0.*", whereas "99designs/http-signatures-php" wants "symfony/http-foundation": "~2.5", to be installed.

I saw your pending PR, but it does not affect symfony/http-foundation's version.

This is a major issue for me. What can I do?

Thank you!

Full output:

```
C:\wamp\bin\php\php5.5.12\php.exe C:\wamp\www\composer.phar require blocktrail/blocktrail-sdk:1.0.4 -n
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - blocktrail/blocktrail-sdk 1.0.4 requires 99designs/http-signatures-guzzlehttp 1.0.0 -> satisfiable by 99designs/http-signatures-guzzlehttp[1.0.0].
    - Installation request for blocktrail/blocktrail-sdk 1.0.4 -> satisfiable by blocktrail/blocktrail-sdk[1.0.4].
    - Conclusion: remove symfony/http-foundation v3.0.4
    - Conclusion: don't install symfony/http-foundation v3.0.4
    - 99designs/http-signatures-guzzlehttp 1.0.0 requires 99designs/http-signatures ~1.1 -> satisfiable by 99designs/http-signatures[v1.1.0, v1.1.1, v1.1.2].
    - 99designs/http-signatures v1.1.0 requires symfony/http-foundation ~2.5 -> satisfiable by symfony/http-foundation[v2.5.0, v2.5.1, v2.5.10, v2.5.11, v2.5.12, v2.5.2, v2.5.3, v2.5.4, v2.5.5, v2.5.6, v2.5.7, v2.5.8, v2.5.9, v2.6.0, v2.6.1, v2.6.10, v2.6.11, v2.6.12, v2.6.13, v2.6.2, v2.6.3, v2.6.4, v2.6.5, v2.6.6, v2.6.7, v2.6.8, v2.6.9, v2.7.0, v2.7.1, v2.7.10, v2.7.11, v2.7.12, v2.7.2, v2.7.3, v2.7.4, v2.7.5, v2.7.6, v2.7.7, v2.7.8, v2.7.9, v2.8.0, v2.8.1, v2.8.2, v2.8.3, v2.8.4, v2.8.5].
    - 99designs/http-signatures v1.1.1 requires symfony/http-foundation ~2.5 -> satisfiable by symfony/http-foundation[v2.5.0, v2.5.1, v2.5.10, v2.5.11, v2.5.12, v2.5.2, v2.5.3, v2.5.4, v2.5.5, v2.5.6, v2.5.7, v2.5.8, v2.5.9, v2.6.0, v2.6.1, v2.6.10, v2.6.11, v2.6.12, v2.6.13, v2.6.2, v2.6.3, v2.6.4, v2.6.5, v2.6.6, v2.6.7, v2.6.8, v2.6.9, v2.7.0, v2.7.1, v2.7.10, v2.7.11, v2.7.12, v2.7.2, v2.7.3, v2.7.4, v2.7.5, v2.7.6, v2.7.7, v2.7.8, v2.7.9, v2.8.0, v2.8.1, v2.8.2, v2.8.3, v2.8.4, v2.8.5].
    - 99designs/http-signatures v1.1.2 requires symfony/http-foundation ~2.5 -> satisfiable by symfony/http-foundation[v2.5.0, v2.5.1, v2.5.10, v2.5.11, v2.5.12, v2.5.2, v2.5.3, v2.5.4, v2.5.5, v2.5.6, v2.5.7, v2.5.8, v2.5.9, v2.6.0, v2.6.1, v2.6.10, v2.6.11, v2.6.12, v2.6.13, v2.6.2, v2.6.3, v2.6.4, v2.6.5, v2.6.6, v2.6.7, v2.6.8, v2.6.9, v2.7.0, v2.7.1, v2.7.10, v2.7.11, v2.7.12, v2.7.2, v2.7.3, v2.7.4, v2.7.5, v2.7.6, v2.7.7, v2.7.8, v2.7.9, v2.8.0, v2.8.1, v2.8.2, v2.8.3, v2.8.4, v2.8.5].
    - Can only install one of: symfony/http-foundation[v2.8.0, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.8.1, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.8.2, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.8.3, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.8.4, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.8.5, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.0, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.1, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.10, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.11, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.12, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.2, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.3, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.4, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.5, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.6, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.7, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.8, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.5.9, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.0, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.1, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.10, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.11, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.12, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.13, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.2, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.3, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.4, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.5, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.6, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.7, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.8, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.6.9, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.0, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.1, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.10, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.11, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.12, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.2, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.3, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.4, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.5, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.6, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.7, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.8, v3.0.4].
    - Can only install one of: symfony/http-foundation[v2.7.9, v3.0.4].
    - Installation request for symfony/http-foundation (locked at v3.0.4) -> satisfiable by symfony/http-foundation[v3.0.4].


Installation failed, reverting ./composer.json to its original content.

```

**Comment by adaur** [comment]
Nevermind, I managed to install the SDK downgrading "symfony/http-foundation" to "~2.8"

Still, I don't like this solution and I'd prefer "99designs/http-signatures-php" to require "2.8.|3.0.", same as Laravel.

**Comment by michaelpittino** [comment]
I have the same problem. Could you tell me how you did downgrade `symfony/http-foundation` to `~2.8`?

**Comment by adaur** [comment]
Yes, sure.

I can't remember exactly, but I think I did a composer require symfony/http-foundation ~2.8.

It added a new line in the require part of my composer.json:

"symfony/http-foundation": "~2.8",

**Comment by rubensayshi** [comment]
I'll try to get the 99design guys to colab on merging my work, otherwise I'll fork it, but I'd rather not

**Comment by rubensayshi** [comment]
fixed in SDK v1.3.4!
